go test fails

```
$ go test
# github.com/nakabonne/ali
./main.go:32:3: Fprintln arg list ends with redundant newline
FAIL    github.com/nakabonne/ali [build failed]
```

--- a/main.go
+++ b/main.go
@@ -34,8 +34,7 @@ Examples:
   ali --duration=10m --rate=100 http://host.xz
 
 Author:
-  Ryo Nakao <ryo@nakao.dev>
-`)
+  Ryo Nakao <ryo@nakao.dev>`)
 	}
 	// Automatically populated by goreleaser during build
 	version = "unversioned"

